src/software/ai/hl/stp/tactic/tactic_test.cpp

```diff
@@ -134,28 +135,30 @@ TEST(TacticTest, test_tactic_restarts_when_set_to_loop_infinitely)
 
 
 
-typedef std::tuple<std::vector<std::pair<RefboxGameState, Point>>, std::vector<AvoidArea>>
-    AvoidAreasTestParams;
+typedef std::tuple<std::vector<std::pair<RefboxGameState, Point>>,
+                   std::set<MotionConstraint>>
+    MotionConstraintsTestParams;
 
 
-AvoidAreasTestParams makeParams(
+MotionConstraintsTestParams makeParams(
     std::vector<std::pair<RefboxGameState, Point>> refbox_and_ball_states,
-    std::vector<AvoidArea> avoid_areas)
+    std::set<MotionConstraint> motion_constraints)
 {
     return std::make_tuple<std::vector<std::pair<RefboxGameState, Point>>,
-                           std::vector<AvoidArea>>(std::move(refbox_and_ball_states),
-                                                   std::move(avoid_areas));
+                           std::set<MotionConstraint>>(std::move(refbox_and_ball_states),
+                                                       std::move(motion_constraints));
 }
 
-class TacticAvoidAreasTest : public ::testing::TestWithParam<AvoidAreasTestParams>
+class TacticMotionConstraintsTest
+    : public ::testing::TestWithParam<MotionConstraintsTestParams>
 {
 };
 
-TEST_P(TacticAvoidAreasTest, test_default_avoid_areas)
+TEST_P(TacticMotionConstraintsTest, test_default_motion_constraints)
```

Comment:
Would it be more appropriate to make this a test for the `MotionConstraintManager`, rather then for the `Tactic` class?

Reply:
There already is the test for that in MotionConstraintManagerTest so I'd agree this test is mostly redundant except for checking that setMotionConstraints works

Reply:
Ah ok, let's remove anything that's redundant then.

Reply:
done see comment above